### DISCUSSION TITLE
D1 mini does not work with EQ3 thermostats, allthough BLE works

### DISCUSSION THREAD

**Discussion by dabepp** [arendst/Tasmota#13945] [Q&A] [answered]
### PROBLEM DESCRIPTION
I had a D1 mini flashed with BLE-Tasmota via the webinstaller. It worked well, I received my Mi thermometers (ATC alternative firmware) and the EQ-3 valves.

Somehow, no clue why, the D1 stopped working so I reinstalled the BLE Tasmota via the webinstaller and since then I can not get the TRVs working. They show up in the tele-message:
`tele/ESPEG/BLE = {"Time":"2021-11-23T11:48:01","BLEDevices":{"total":7,...,"001Axxxxxxxx":{"i":4,"r":-94,"a":"EQ3OG"}, ...`
But the TRV commands like TRV devlist
`11:40:46.919 CMD: trv devlist`
which should list the received TRVs is giving me this answer:
`11:40:46.935 MQT: stat/ESPEG/RESULT = {"Command":"Unknown"}`
BLE is activated, MQTT is giving output about the ATCs, I have put this in autoexec.bat:

BLEAlias A4C138xxxxxx=ATCWohnzimmer
...
BLEAlias A4C138xxxxxx=ATCWaschkeller
BLEAlias A4C138xxxxxx=ATCReserve
BLEAlias 001Axxxxxxxx=EQ3Keller
BLEAlias 001Axxxxxxxx=EQ3OG
BLEAddrFilter 0
MI32Option6 1
bleenableunsaved 1
trvHideFailedPoll 1
trvMinRSSI -95

What am I missing?

This is the Tasmota basic Info:

Program Version:	10.0.0.3(bluetooth)
Build Date & Time:  2021-11-17T13:45:44
Core/SDK Version:	2_0_1/v4.4-dev-3631-g7033e3b57

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [ ] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [X] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [X] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [X] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [X] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [X] Device used (e.g., Sonoff Basic): WEMOS D1 mini ESP32
- [X] Tasmota binary firmware version number used: 10.0.0
  - [X] Pre-compiled
  - [ ] Self-compiled
- [X] Flashing tools used: Web Installer
- [] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:


```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [ ] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:


```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
_Steps to reproduce the behavior:_

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by sfromis** [comment, marked as answer]
If you're on the Discord server for Tasmota, you can [see](https://discord.com/channels/479389167382691863/796060106776117288/917517119959220284) that @barbudor is fixing that, wanting his build to be tested "in real life".

**Reply by barbudor** [reply]
EQ3 was temporarily disabled due to a compile error with new Arduino Core in Tasmota 10.0.0.2
The  fix has been merge a few minutes ago and should be available in precompiled binary within 1 hour

**Reply by dabepp** [reply]
This makes sense now! Thank you!
